dart/lib/src/utils.dart

```diff
@@ -0,0 +1,5 @@
+bool indexOfMatch(String string, Pattern pattern) =>
+    string.indexOf(pattern) >= 0;
+
+bool indexOfNotMatch(String string, Pattern pattern) =>
```

Comment:
This doesn't look like it's used anywhere?

Reply:
Yep, sorry.